src/messages/de.json

```diff
@@ -3,6 +3,12 @@
     "title": "Open Data Dashboard",
     "description": "Visualisierungen verschiedener Datensätze der Stadt Karlsruhe"
   },
+  "Visualization": {
+    "dataEmptyTitle": "Ressource beinhaltet keine Daten!",
+    "dataEmptyMessage": "Endpunkt \"{source}\" für Ressource \"{name}\" mit id {id} hat keine/leere Daten zurückgegeben. Melden sie dieses Problem dem Administrator dieser Seite.",
```

Comment:
This is a matter of taste.

In my opinion, the error message output to the user doesn't need to be this verbose. The content you have in the `errorTitle` would be enough. In the subtitle just put the second sentence (`Falls diese Problem weiterhin besteht ...`).

The detailed output (contents of first subtitle sentence) we could log as error to the console together with the actual exception (which is already logged I think).

Reply:
Good point